includes/Elements/Team_Member.php

```diff
@@ -910,9 +913,16 @@ protected function register_controls() {
 	protected function render( ) {
 
         $settings = $this->get_settings_for_display();
-		$team_member_image = $this->get_settings( 'eael_team_member_image' );
+		$team_member_image = $settings['eael_team_member_image'];
```

Comment:
Even though we have this value all the time, but doing empty check will be good too.
```
$team_member_image = $settings['eael_team_member_image'] ?? '';
```